src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -757,6 +540,204 @@ namespace EcoRoofManager {
+                            thisEcoRoof->CumIrrigation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Runoff Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumRunoff,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Evapotranspiration Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumET,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Precipitation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentPrecipitation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Irrigation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentIrrigation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Runoff Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentRunoff,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Evapotranspiration Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentET,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+    }
+
+    void initEcoRoof(EnergyPlusData &state, int const SurfNum, int const ConstrNum)
+    {
+        // Using/Aliasing
+        using namespace DataEnvironment;
+
+        auto &thisMat = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1));
+        auto &thisEcoRoof = state.dataEcoRoofMgr;
```

Comment:
I was going to say this is a weird `using`, but I think the weird part is that it appears we can only have one eco roof in a simulation file.  We can have multiple buildings, but only one eco roof.  Seems problematic, but nothing to worry about here.